libs/openFrameworks/gl/ofGLProgrammableRenderer.cpp

```diff
@@ -1315,12 +1321,46 @@ void ofGLProgrammableRenderer::bind(const ofFbo & fbo, bool setupPerspective){
 }
```

Comment:
and the fbo is bound in the renderer upon bind(fbo), as the renderer needs to keep track of the fbo binding state in its local scope.